docs/Rules/MA0090.md

```diff
@@ -1 +1,139 @@
 # MA0090 - Remove empty else/finally block
+
+This rule detects empty `else` and `finally` blocks that contain no executable code and suggests removing them to improve code readability and reduce clutter. Empty blocks serve no functional purpose and can make code harder to read by adding unnecessary visual noise.
+
+## Non-compliant code
```

Comment:
The structure doesn't match other rule documentation. Most of the time a code sample with inline comments are sufficient.